config/application.rb

```diff
@@ -20,5 +20,7 @@ class Application < Rails::Application
     config.govuk_notify_templates = config_for(
       :govuk_notify_templates, env: ENV.fetch('GOVUK_NOTIFY_ENV', 'development')
     ).symbolize_keys
+
+    config.eager_load_paths << Rails.root.join('lib/omniauth')
```

Comment:
**not a blocker** FYI, I came across this previously in the project, it seems Rails has removed autoload from `lib` directories, and they recommend having this sort of functionality in `app/lib` 🤷‍♂️ : https://stackoverflow.com/questions/38198668/rails-5-load-lib-files-in-production